.gitignore

```diff
@@ -2,6 +2,7 @@ coverage
 dist
 node_modules
 npm-error.log
-
+.DS_Store
```

Comment:
We will have tiny collision with the other pending PR -- we both added .DS_store to .gitignore. As they say, "great minds think alike."